modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -14,39 +14,46 @@ parameters.
 apiVersion: v1
 baseDomain: example.com <1>
 compute:
-- name: worker
-  replicas: 0 <2>
+- name: worker <2>
+  replicas: 0 <3>
 controlPlane:
-  name: master
-  replicas: 3 <3>
+  name: master <2>
+  replicas: 3 <4>
 metadata:
-  name: test <4>
+  name: test <5>
 networking:
   clusterNetworks:
-  - cidr: 10.128.0.0/14 <5>
-    hostPrefix: 23 <6>
+  - cidr: 10.128.0.0/14 <6>
+    hostPrefix: 23 <7>
   networkType: OpenShiftSDN
-  serviceNetwork: <7>
+  serviceNetwork: <8>
   - 172.30.0.0/16
 platform:
   none: {}
-pullSecret: '{"auths": ...}' <8>
-sshKey: 'ssh-ed25519 AAAA...' <9>
+pullSecret: '{"auths": ...}' <9>
+sshKey: 'ssh-ed25519 AAAA...' <10>
 ----
 <1> The base domain of the cluster. All DNS records must be sub-domains of this
 base and include the cluster name.
-<2> You must set the number of workers to `0` in this file. Remember that you
+<2> The `controlPlane` section is a singlet, and the `compute` section is an
+array. Because the sections have different definitions, the first line of the
+`compute` section must begin with a hyphen, `-`, and the first line of the
+`controlPlane` section must not. Although both sections currently define a
+single machine pool, it is possible that future versions of {product-title}
+will support defining multiple compute pools during installation. Only one
+control plane pool is used.
+<3> You must set the number of workers to `0` in this file. Remember that you
 must manually deploy at least one compute, or worker, machine in the cluster.
-<3> The number of control plane machines that you add to the cluster. Because
+<4> The number of control plane machines that you add to the cluster. Because
 the cluster uses this values as the number of etcd endpoints in the cluster, the
 value must match the number of control plane machines that you deploy.
-<4> The cluster name.
-<5> A block of IP addresses from which pod IPs are allocated.
-<6> The subnet prefix length to assign to each individual node. For example, if
+<5> The cluster name.
+<6> A block of IP addresses from which pod IPs are allocated.
+<7> The subnet prefix length to assign to each individual node. For example, if
 `hostPrefix` is set to `23`, then each node is assigned a `/23` subnet out of
 the given `cidr`, which allows for 510 (2^(32 - 23) - 2) pod IPs addresses.
-<7> The IP address pool to use for service IP addresses. You can enter only
+<8> The IP address pool to use for service IP addresses. You can enter only
 one IP address pool.
-<8> The pull secret that provides components in the cluster access to images for
+<9> The pull secret that provides components in the cluster access to images for
```

Comment:
@kalexand-rh  @wking my 0.02 $: for folks who are new on v4 or those coming from v3, i think will be useful to reword it to explicitly say what this pullSecret is, something like 

```
The pull secret that provides components in the cluster access to container images for {product-title} components
```
i think will do.

I'd even go and maybe extend the pullSecret glossary to say that is nothing but the login credentials to a docker private registry which is holding the container images

Reply:
"images" -> "container images" is fine with me.  And that's the main pull-secret goal, but it also authorizes you to telemetry, and maybe other things I'm not aware of.  So in anything as global as a glossary, I'd probably say that it's about authenticating your cluster as belonging to you for services provided by the included authorities, including quay.io for serving container images.

Reply:
@wking, I updated the pull secret description. What do you think?

Reply:
Looks great

Reply:
looks great, thanks @kalexand-rh @wking !